cabal.project

```diff
@@ -0,0 +1,12 @@
+packages:
+  ./.
+  examples/bearriver/
```

Comment:
Wouldn't it be better to use a wildcard `examples/*/*.cabal examples/*/*/*.cabal`? That way, new examples get automatically CI'd.